dcrd.go

```diff
@@ -44,6 +44,8 @@ func dcrdMain(serverChan chan<- *server) error {
 
 	// Show version at startup.
 	dcrdLog.Infof("Version %s", version())
+	// Show path location
+	dcrdLog.Infof("Work Path: %s", cfg.DcrdHomeDir)
```

Comment:
Also, I think this should be Debug, not Info (since I would prefer info stay cleaner.